frontend/src/Hooks/useEditWorkshop.ts

```diff
@@ -72,7 +70,6 @@ export default function useEditWorkshop(props: EditWorkshopFormProps){
                         inventory: components
                     })
                         .then(r => r.data)
```

Comment:
This line is now useless I think, you don't seem to use the response anymore

Reply:
yep